ui/tooltip.js

```diff
@@ -331,7 +337,7 @@ return $.widget( "ui.tooltip", {
 		// tooltips will handle this in destroy.
 		if ( target[ 0 ] !== this.element[ 0 ] ) {
 			events.remove = function() {
-				this._removeTooltip( tooltip );
+				this._removeTooltip( this._find( target ).tooltip );
```

Comment:
This line was changed from `this._find( target )` to `this._find( target ).tooltip` because we now store structured data, not just the tooltip element.